measures/openstudio_results/resources/os_lib_reporting.rb

```diff
@@ -1469,7 +1481,12 @@ def self.envelope_section_section(model, sqlFile, runner, name_only = false)
           if use_old_gem_code
             shgc = construction_root.calculated_solar_heat_gain_coefficient
           else
-            shgc = std.construction_calculated_solar_heat_gain_coefficient(construction_root)
+            if std.respond_to?('construction_calculated_solar_heat_gain_coefficient')
+              shgc = std.construction_calculated_solar_heat_gain_coefficient(construction_root)
+            else
+              # 0.6.0+
+              shgc = OpenstudioStandards::Constructions.construction_get_solar_transmittance(surface_construction)
+            end
```

Comment:
I started porting some of the code in openstudio_results to use newer methods and avoid sections breaking because NoMethodError or Optional not initialized, but there would be more to do to get it perfect.

@DavidGoldwasser  as you already know, even if this specific measure inside this repo is very outdated, the most current on openstudio-common-measures-gem will also need some of the same treatment